src/timer.rs

```diff
@@ -77,3 +77,17 @@ pub trait CountDown {
 
 /// Marker trait that indicates that a timer is periodic
 pub trait Periodic {}
+
+/// Trait for cancelable countdowns.
+pub trait Cancel : CountDown {
+    /// Error returned when a countdown can't be canceled.
```

Comment:
Typo. Should be "cancel**l**ed" (one "l" missing).

Reply:
Yeah I had to look this up, too. Decided to go with American English since most internationally used words in computer science and engineering use their AE spelling in favor of BE.